pkg/apis/etcd/v1beta2/cluster.go

```diff
@@ -162,6 +162,11 @@ type PodPolicy struct {
 	// '.cluster.local'.
 	// The default is to not set a cluster domain explicitly.
 	ClusterDomain string `json:"ClusterDomain"`
+
+	// curl init container image. default is tutum/curl.
+	// pulling tutum/curl:latest requires external access.
+	// More info: https://github.com/coreos/etcd-operator/issues/1933
+	CurlImage string `json:"curlImage,omitempty"`
```

Comment:
Better to add this to `EtcdRestore` struct

Reply:
I thought the `busyboxImage` parameter is used for all etcd cluster pods and not just etcd restore https://github.com/coreos/etcd-operator/blob/b2a63ce924602be64a156361f3ee85498843026a/pkg/util/k8sutil/k8sutil.go#L374